WooCommerce/Classes/ViewRelated/Products/UpdateInventory/UpdateProductInventoryView.swift

```diff
@@ -88,9 +90,14 @@ struct UpdateProductInventoryView: View {
                         .disabled(!viewModel.enableQuantityButton)
                         .padding(.bottom, Layout.mediumSpacing)
 
-                        Button(Localization.viewProductDetailsButtonTitle) {}
+                        Button(Localization.viewProductDetailsButtonTitle) {
+                            isPresentingDetailsView = true
+                        }
                             .buttonStyle(SecondaryButtonStyle())
                             .padding(.bottom)
+                            .sheet(isPresented: $isPresentingDetailsView) {
+                                viewModel.productDetailsView()
+                            }
```

Comment:
Some off-padding here :D 

```suggestion
                        Button(Localization.viewProductDetailsButtonTitle) {
                            isPresentingDetailsView = true
                        }
                        .buttonStyle(SecondaryButtonStyle())
                        .padding(.bottom)
                        .sheet(isPresented: $isPresentingDetailsView) {
                            viewModel.productDetailsView()
                        }
```